templates/Rserv.conf.j2

```diff
@@ -0,0 +1,12 @@
+workdir {{ rserve.workdir }}
+pwdfile /etc/Rserv.pwd
+remote enable
+auth required
```

Comment:
For my purposes I'm fine with making this `auth disable`. Please see my related comment at https://github.com/IQSS/dataverse-ansible/pull/87#issuecomment-510646442

I'm also fine with merging as-is because I got Data Explorer working and I have to do some manual config anyway:

- Accept the invalid (self-signed) cert at, for example, https://ec2-54-208-71-42.compute-1.amazonaws.com
- Use the hostname above, changing http to https in as the `dataverse.siteUrl` in /usr/local/glassfish4/glassfish/domains/domain1/config/domain.xml
- Add `ProxyPass / ajp://localhost:8009/` to `/etc/httpd/conf.d/ssl.conf`
- Disable rserve auth per above and restart rserve.

To be clear to anyone reading this in the future, I'm just using this for testing and demo purposes on throw away EC2 instances so I'm not particularly worried about rserve security. For production tips on security, please see http://guides.dataverse.org/en/4.15/installation/prerequisites.html#rserve